### ISSUE TITLE
What is Link between MN Login and the Running MN?

### ISSUE THREAD

**Issue by presman** [XinFinOrg/XinFin-Node#12]
Hello Support Team - Prior guidance is to setup (2) Wallets for the Master Node:
(1) Private wallet to hold the 10M XDC tokens, this will be LOCKED for 30 days.
(2) Public wallet to receive MN staking rewards into.  (Put this into coinbase.txt file)

Question (1):  Is the MN Login private key for the Private 10M account or Public rewards account?

Question (2):  Only 3 fields appear:  Public key Addr for login PK, XDC Balance, KYC false.  I do not see any way to link the private account to the public account.  How do we know the MN rewards wallet in coinbase.txt file is linked to the Private 10M XDC 30 day Locked Wallet?

Note:  Possible bug - the MN Login displays a public key in all lower case.  It is not displaying the upper and lower case characters in the key.

Thanks again for all of your help!
Preston

**Comment by shrikantgoswami** [comment]
Answer (1): Private 10M account.

Answer (2): After you login & apply for masternode with the 10M stake you should see your node as a  candidate here http://xinfin.network/#masternode 

Also please note you don't need to edit the coinbase.txt file, it's generated for you to quickly find & copy the coinbase address when you need to apply for masternode.

> Note: Possible bug - the MN Login displays a public key in all lower case. It is not displaying the upper and lower case characters in the key.

Noted.

Thanks

**Comment by presman** [comment]
Hello - So after the Login to MN with test XDC 10M Private Key, I do not see any other action on this screen.  What actions do I need to take to "apply for masternode" to get listed on the master node candidates screen?

Also, This login screen does not ask for my MN names, MN IP addresss or coinbase.txt public rewards address.  Therefore, I do not see any way to link this private key to the running MN that has a different key to receive rewards.

So basically,

(1) I don't know "how to apply"

(2) How to submit and pass KYC (or is this just for when we go  live)?

(3) How to link this private key login to my running MN (name, IP or coinbase.txt addr)

Thank you again for your help. 
Preston
